### DISCUSSION TITLE
Retry Multiplier and Increment conflicts?

### DISCUSSION THREAD

**Discussion by globalflea** [serverlessworkflow/specification#555] [Q&A]
In Retry Definition, spec says:
increment - Static duration which will be added to the delay between successive retries 
multiplier - Float value by which the delay is multiplied before each attempt

in the first example provided delay is "PT10S" and increment is "PT2S" => 10, 12, 14, 16
in the second example provided delay is "PT10S" and multiplier is "2 => 10, 20, 40, 80

What happens when both increment & multiplier are defined at the same time as with the above values - should the 2nd retry by 12 or 20 ?

**Comment by tsurdilo** [comment]
Idea in that case is to increment, then multiply. Need issue in spec repo to add that to text. Ty
